code/config.py

```diff
@@ -1,120 +1,408 @@
-# DeepSlide
-# Jason Wei, Behnaz Abdollahi, Saeed Hassanpour
+"""
+DeepSlide
+Contains all hyperparameters for the entire repository.
 
-# This file contains all hyperparameters for the entire repository.
+Authors: Jason Wei, Behnaz Abdollahi, Saeed Hassanpour
 
-import utils
-from utils import *
+Last Modified: November 30, 2019 (Joseph DiPalma)
+"""
+
+import argparse
+from pathlib import Path
+
+import torch
+
+from compute_stats import compute_stats
+from utils import (get_classes, get_log_csv_name)
+
+# Source: https://stackoverflow.com/questions/12151306/argparse-way-to-include-default-values-in-help
+parser = argparse.ArgumentParser(
+    description="DeepSlide",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
 ###########################################
 #               USER INPUTS               #
 ###########################################
+# Input folders for training images.
+# Must contain subfolders of images labelled by class.
+# If your two classes are 'a' and 'n', you must have a/*.jpg with the images in class a and
+# n/*.jpg with the images in class n.
+parser.add_argument(
+    "--all-wsi",
+    type=Path,
+    default=Path("all_wsi"),
+    help="Location of the WSI organized in subfolders by class")
+# For splitting into validation set.
+parser.add_argument("--val-wsi-per-class",
```

Comment:
You can just changed this to `--val_wsi_per_class`.
It should be in snake case unless you have a clear reason why it shouldnt

Reply:
@jasonwei20 I'll switch to snake case.